contracts/deployless/NFTGetter.sol

```diff
@@ -58,17 +58,21 @@ contract NFTGetter is Simulation {
       uint total;
       for (uint i = 0; i != tokenIds.length; i++) {
         if (total == limit) break;
-        if (collection.ownerOf(tokenIds[i]) == address(account)) {
-          total++;
-        }
+        try collection.ownerOf(tokenIds[i]) returns (address ownerOfCurrentToken) {
```

Comment:
added i little more detailed explanation

Reply:
// catching the call as we can tolerate errors here because:
// - on nft mint the token does not exist before the simulation and ownerOf fails
// - on nft burn the token does not exist after the simulation and ownerOf fails